src/coord/rmr/conn.c

```diff
@@ -6,6 +6,8 @@
 
 #include "conn.h"
 #include "reply.h"
+#include "../config.h"
```

Comment:
can you fix this include line?
I don't think we generally use relative path includes in our code.